parity-clib/examples/cpp/main.cpp

```diff
@@ -49,58 +48,38 @@ const std::vector<std::string> ws_subscriptions {
 	"{\"method\":\"eth_subscribe\",\"params\":[\"newHeads\"],\"id\":1,\"jsonrpc\":\"2.0\"}"
 };
 
+std::atomic<int> callback_counter;
```

Comment:
`int` is probably the wrong type here.  I suspect that `std::int64_t` is the correct one.